src/mc-fle2-find-range-payload-private-v2.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MC_FLE2_FIND_RANGE_PAYLOAD_PRIVATE_V2_H
+#define MC_FLE2_FIND_RANGE_PAYLOAD_PRIVATE_V2_H
+
+#include "mongocrypt.h"
+#include "mongocrypt-private.h"
+#include "mongocrypt-buffer-private.h"
+
+#include "mc-array-private.h"
+#include "mc-fle2-range-operator-private.h"
+
+/** FLE2FindRangePayloadEdgesInfoV2 represents the token information for a range
+ * find query. It is encoded inside an FLE2FindRangePayloadV2.
+ */
+typedef struct {
+   mc_array_t edgeFindTokenSetArray;           // g
+   int64_t maxContentionCounter;               // cm
+} mc_FLE2FindRangePayloadEdgesInfoV2_t;
+
+/**
+ * FLE2FindRangePayloadV2 represents an FLE2 payload of a range indexed field to
+ * query. It is created client side.
+ *
+ * FLE2FindRangePayloadV2 has the following data layout:
+ *
+ * struct {
+ *   uint8_t fle_blob_subtype = 10;
```

Comment:
```suggestion
 *   uint8_t fle_blob_subtype = 13;
```

Copied from: https://github.com/mongodb/mongo/blob/dbfc672d325ee924f84c8856f490522c57635acc/src/mongo/crypto/fle_field_schema.idl#L66